server/protocols/webadmin/webadmin-cassandra/src/main/java/org/apache/james/webadmin/routes/CassandraMigrationRoutes.java

```diff
@@ -144,7 +144,7 @@ public Object upgradeToVersion(Request request, Response response) {
         CassandraVersionRequest cassandraVersionRequest = CassandraVersionRequest.parse(request.body());
         Task migration = cassandraMigrationService.upgradeToVersion(cassandraVersionRequest.getValue());
         TaskId taskId = taskManager.submit(migration);
-        return TaskIdDto.from(taskId);
+        return TaskIdDto.respond(response, taskId);
```

Comment:
why a DTO is supposed to have a method to interact with a response?
It doesn't look like a DTO anymore

Reply:
See https://github.com/linagora/james-project/pull/2990/commits/c1f4e0cba9f0fb66ef46a02700eb2eadb3e53e3d

I will remove the part related to your remark from this work.